<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QWidget" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#FFFFFF;
border:1px solid #cdcdcd;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QWidget
{
	background:transparent;
	border:none;
}

/*background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #F1F6FF , stop:1 #E1EEFF);*/
#widget_title{
background-color:rgb(223,190,111);
}

QLabel{
border:none;
	background:transparent;

}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1,0,0,0,0,0">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/icons/images/taiji.png);
background:transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>87</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight:700;
color:#333333;
font-size:24px;
line-height:16px;</string>
        </property>
        <property name="text">
         <string>杏林医疗信息系统</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_time">
        <property name="styleSheet">
         <string notr="true">font-weight:700;
color:#333333;
font-size:20px;
line-height:16px;
border:none;
padding-left:512px;</string>
        </property>
        <property name="text">
         <string>2023-10-24 11:31:06</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_user">
        <property name="minimumSize">
         <size>
          <width>56</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #3E5B82;
border-radius: 2px;
font-size: 16px;
font-weight: 500;
color: #E6F1FF;</string>
        </property>
        <property name="text">
         <string>张飞</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QComboBox
{
	background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #F1F6FF , stop:1 #E1EEFF);
	border-radius: 2px;
	border: 1px solid #E9EEF5;
	font-size: 18px;
	font-weight: 400;
	color: #1D2129;
	line-height: 22px;
	padding-left:8px;
}
QComboBox::down-arrow
{
	border-image:url(&quot;:/icons/images/icons_combobox_down_arrow.png&quot;);
}
QComboBox::drop-down
{
	border:none;
	background:transparent;
	padding-right:6px;
}

/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    outline: 0px solid gray;   
    border: none;   
    background-color: #FFFFFF;   
    selection-background-color: #40a9ff;   
	font-size: 14px;
	font-weight: 400;
	color: #1D2129;
	line-height: 22px;
	border-radius:3px;
	margin-left:4px;
}

/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
    height: 40px;   /* 项的高度 */
}

/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
   color:#FFFFFF;
   background-color: #1479FF;   /* 整个下拉窗体越过每项的背景色 */
}

/* QComboBox中的垂直滚动条 */
QComboBox QAbstractScrollArea QScrollBar:vertical {
    border:none;
	background:transparent;
}

QComboBox QAbstractScrollArea QScrollBar::handle:vertical {
	width: 8px;
    background-color:;   /* 空白区域的背景色*/
	border-radius:4px;
    background: #C9CDD4;   /* 小方块的背景色深灰lightblue */
}</string>
        </property>
        <item>
         <property name="text">
          <string>门诊医生工作站</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>住院医生工作站</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/icons/images/logout.png);
border:none;
background:transparent;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="styleSheet">
         <string notr="true">font-weight:700;
color:#333333;
font-size:20px;
line-height:16px;
background:transparent;
border:none;</string>
        </property>
        <property name="text">
         <string>退出系统</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
